src/react-components/ui-root.js

```diff
@@ -638,36 +670,37 @@ class UIRoot extends Component {
     this.props.scene.emit("add_media", media);
   };
 
-  closeDialog = () => {
-    this.setState({ dialog: null });
+  closeDialog = success => {
+    if (this.state.dialog) {
+      this.setState({ dialog: null });
+    } else {
+      // If dialog was successful (eg user hit "OK") then move forward in history, o/w go back.
+      //
+      // This makes it so if you create an object, back will re-show the create object dialog,
+      // but if you cancel, it will not.
+      if (success) {
+        this.props.history.push("/");
+      } else {
+        this.props.history.goBack();
```

Comment:
This means I will be able to go "forward" into the dialog, not sure that is desirable.